nettoyage.py

```diff
@@ -89,33 +91,119 @@ def check_reference_files():
     """
     path_data = conf_data["path_to_data"]
 
-    useless_keys = ["path_to_project", "path_to_data", "path_to_cache", "cache_bdd_insee",
+    l_key_useless = ["path_to_project", "path_to_data", "path_to_cache", "cache_bdd_insee",
                      "cache_not_in_bdd_insee",
                      "cache_bdd_legale",
                      "cache_not_in_bdd_legale"]
 
     path = os.path.join(os.getcwd(), path_data)
     for key in list(conf_data.keys()):
-        if key not in useless_keys:
-
+        if key not in l_key_useless:
             logger.info(f'Test du fichier {conf_data[key]}')
             mask = os.path.exists(os.path.join(path, conf_data[key]))
             if not mask:
                 logger.error(f"Le fichier {conf_data[key]} n'existe pas")
                 raise ValueError(f"Le fichier data: {conf_data[key]} n'a pas été trouvé")
 
 
-def manage_titulaires(df: pd.DataFrame):
-    # Ecriture dans les logs
-    logger.info(f"Nombre de marché sans titulaires: {sum(df['titulaires'].isnull())}. "
-                f"Remplacé par la valeur du concessionnaire")
-    logger.info(f"Nombre de marché sans montant: {sum(df['montant'].isnull())}. Remplacé par la valeur globale")
-    logger.info(f"Nombre de marché sans identifiant acheteur: {sum(df['acheteur.id'].isnull())}. "
-                f"Remplacé par l'identifiant de l'autorité concédante")
-    logger.info(f"Nombre de marché sans nom d'acheteur: {sum(df['acheteur.nom'].isnull())}. "
-                f"Remplacé par le nom de l'autorité concédante")
-
-    # Gestion différences concessionnaires / titulaires
+def found_values_in_dic(x, name:str):
+    if type(x) == dict:
+        if name in x.keys():
+            return x[name]
+    else :
+        return None
```

Comment:
comparer performance avec 
try:
   return x.get(name)
except:
   return None

Reply:
"in most of the cases when there is very little possibility of a condition being false, we should use try-except as they are much faster and efficient."

Reply:
avec try except c'est en effet plus rapide de 14% environ.